docs/logging-format.md

```diff
@@ -228,6 +228,45 @@ and TLS probe points.
 | `tls::key_exchange_algorithm` | uint16     | Key exchange mode: ECDHE(0), DHE(1), PSK(2), ECDHE-PSK(3), DHE-PSK(4)                            |
 | `tls::group`                  | uint16     | Groups used in the handshake (as in IANA [registry][iana-tls-supported-groups])                  |
 
+##### SSH context names
+
+| name                   | description                            |
+|------------------------|----------------------------------------|
+| `ssh::handshake_client`| SSH handshake for client               |
+| `ssh::handshake_server`| SSH handshake for server               |
+| `ssh::client_key`      | SSH client key signature/verification  |
+| `ssh::server_key`      | SSH server key signature/verification  |
+| `ssh::key_exchange`    | SSH key exchange                       |
+
+Example of SSH context tree:
+
+- `ssh::handshake_client`
+  - `ssh::ident_string` = `SSH-2.0-OpenSSH_8.8`
+  - `ssh::client key`
```

Comment:
underscore instead of space
```suggestion
  - `ssh::client_key`
```
I assume the events should come in order they happen during the ssh session so the client key operation should happen only after the key_exchange is finished.

The hostkey verification happens as part of the key exchange though so it is a question whether it should be part of that context or somehow separately.

In any case, this is a context so it should have again some events attached for example:
```suggestion
  - `ssh::client_key`
    - `ssh::key_algorithm` = `rsa-sha2-256`
    - `ssh::rsa_bits` = `2048`
```